.travis.yml

```diff
@@ -51,9 +51,9 @@ install:
   - pip list
 script:
   - if [[ "${TRAVIS_OS_NAME}" == "osx" ]]; then
```

Comment:
TODO: check if this is still needed with pytest.

Reply:
We could try to do coverage for both osx and for linux, but I don't think it would be any different. Coverage shouldn't change much from system to system.  Either way I don't think nose was causing this choice, so pytest shouldn't make us change. I feel we can leave coverage turned off for osx to simplify and also to reduce an already (7x) longer execution time on the osx platform.